peachjam/urls.py

```diff
@@ -257,6 +258,12 @@
     ),
     # django-markdown-editor
     path("martor/", include("martor.urls")),
+    # Get a specific media file for a document
+    re_path(
+        r"documents/(?P<document_id>[0-9]+)/media/(?P<filename>.*)$",
```

Comment:
this must be the FRBR URI version of this url, not the document id version. Otherwise the media won't load correctly. See how it is described in the original issue.